rfcs/0004-vello-intg-reorg.md

```diff
@@ -0,0 +1,59 @@
+# Feature Name: `vello-intg-reorg`
+
+## Summary
+
+This RFC has a simple, targeted goal to move integrations (`vello/integrations/vello_svg`) out of the `vello` crate, similar to `velato`. This would be implemented **after** 1.0 of vello.
+
+In addition, I propose deprecation of `vello_svg` and `velato` in favor of bringing [`vello-svg`](https://github.com/vectorgameexperts/vello-svg) and [`vellottie`](https://github.com/vectorgameexperts/vellottie) into the GitHub linebender organization as the official integrations.
+
+Lastly, the vello docs would make a mention of integrations and where to find them in their respective repos.
+
+## Motivation
+
+- Simplify the publish cycle for `vello` - Only 1 crate (vello) to crates.io in CI/CD.
+- Standard treatment of integrations (currently we have vello_svg in the vello crate, but velato as a standalone)
+- Keep PRs focused, rather than needing changes to `vello_svg`
+
+In favor of deprecating `vello_svg` and `velato` for `vello-svg` and `vellottie`:
+
+- vello-svg is a 1:1 mirror, but has additions like a [nursery](https://vectorgameexperts.github.io/vello-svg/) and tests.
+- vellottie is nearly a 1:1 mirror with velato's runtime, except it might have some cleanup and rendering fixes. It has 2 parsers, a fast, new serde parser, and a step-through parser used on parse error to identify lottie file issues with granularity. It also has a [nursery](https://vectorgameexperts.github.io/vellottie) and tests.
+
+## User-facing explanation
+
+- Need SVG? Go to the vello-svg README.
+- Need Lottie? Go to the vellottie README.
```

Comment:
This is *way* too short.

As an outside user, if I found a paragraph in Vello's repo that said "Go to the vello-svg README" and nothing else, I would assume the project was amateur or a stub.

Reply:
This was short, agreed, but I thought conveyed the intention.

The vello repo would just point to the appropriate documentation relative to the integration the user may be looking for; an advertisement.